packages/IMAPClient-Core.package/ICCommand.class/class/fetchFlagsWithUIDFrom.to..st

```diff
@@ -0,0 +1,4 @@
+commands
+fetchFlagsWithUIDFrom: aBeginUID to: anEndUID
```

Comment:
Seems to be a misleading name, maybe use `fetchFlagsForUIDsFrom: aBeginUID to: anEndUID`